python/apps/taiga/src/taiga/workflows/events/__init__.py

```diff
@@ -28,3 +33,13 @@ async def emit_event_when_workflow_status_is_updated(project: Project, workflow_
         type=UPDATE_WORKFLOW_STATUS,
         content=UpdateWorkflowStatusContent(workflow_status=workflow_status),
     )
+
+
+async def emit_event_when_workflow_status_is_deleted(project: Project, workflow_status: WorkflowStatus) -> None:
```

Comment:
don't they need to know the move_to_status to move the stories?

Reply:
Yes they do. I've just uploaded the latest code reflecting this last change.